pkg/plugin/v2/plugin.go

```diff
@@ -23,7 +23,7 @@ import (
 
 const (
 	pluginName    = "go" + plugin.DefaultNameQualifier
-	pluginVersion = "v2.0.0"
+	pluginVersion = "v2.1.0"
```

Comment:
IHMO it shoud be a 3.0.0 plugin. 
So, I am trying to figure out how to do that. 

c/c @estroz

Reply:
It seems we need to discuss how scaffolding changes affect versions more, but a major version bump for any scaffolding change makes sense since they’re all technically breaking changes.